README.md

```diff
@@ -1,6 +1,14 @@
 # Celestia Node
```

Comment:
Can we add the usual badges here as well?
- godoc 
- code coverage
- lastest release
- etc

Reply:
@Bidon15 I added release tag and godoc badges, do you know how to do codecov? I know codecov already has access to celestiaorg IIRC

Reply:
Codecov can also happen in a separate PR imo.

Reply:
I see that Viet already started: https://github.com/celestiaorg/celestia-node/pull/564